### PR TITLE
Fixes for inclusion

### PR DESCRIPTION
**Pull request by estebanlm**

### CODE REVIEW

File: src/NewTools-DocumentBrowser/MicDocumentBrowserStyleContributor.class.st [hunk 1]
```diff
@@ -0,0 +1,16 @@
+"
+A style contributor for the document browser, it adds specific styles to be used by `MicDocumentBrowser` in the context of the `StPharoApplication`.
+"
+Class {
+	#name : #MicDocumentBrowserStyleContributor,
```
Comment (demarey) [inline src/NewTools-DocumentBrowser/MicDocumentBrowserStyleContributor.class.st:5]: I do not see where this class is used?

File: src/Microdown-RichTextPresenter/MicMorphicTextAdapter.class.st [hunk 2]
```diff
@@ -56,8 +90,14 @@ MicMorphicTextAdapter >> selectionInterval [
...
 MicMorphicTextAdapter >> selectionInterval: anInterval [
 	"called by presenter to set selection in morph"
-	self widgetDo: [ :w | w setTextAreaSelection: anInterval ].
-	
+
+	self widgetDo: [ :w | w setTextAreaSelection: anInterval ]
+]
+
+{ #category : #factory }
+MicMorphicTextAdapter >> setEnabled: aBoolean to: aWidget [
```
Comment (demarey) [inline src/Microdown-RichTextPresenter/MicMorphicTextAdapter.class.st:98]: I do not see the value of this method.
`aWidget enabled: aBoolean` is more clear

File: src/NewTools-DocumentBrowser/MicDocumentBrowser.class.st [hunk 3]
```diff
@@ -313,30 +316,28 @@ MicDocumentBrowser >> initializeButtons [
 	spButtonBar := self instantiate: MicDocumentBrowserButtonBar.
 	spButtonBar
 		left: (spButtonBar newCheckBox
-				 label: 'Show List';
-				 state: layoutModel showDocumentList;
-				 whenChangedDo: [ :val | self guiSetListShowing: val ]);
+			label: 'Show List';
+			state: layoutModel showDocumentList;
+			whenChangedDo: [ :val | self guiSetListShowing: val ]);
```
Comment (demarey) [inline src/NewTools-DocumentBrowser/MicDocumentBrowser.class.st:321]: Do we keep `guiXxx` method  prefix?

File: src/NewTools-DocumentBrowser/MicDocumentBrowserStyleContributor.class.st (hunk 1 above)
Comment (estebanlm) [reply src/NewTools-DocumentBrowser/MicDocumentBrowserStyleContributor.class.st:5]: is a contributor which is the way a tool part of new-tools has to use to add tool-specific styles to  StPharoApplication

File: src/Microdown-RichTextPresenter/MicMorphicTextAdapter.class.st (hunk 2 above)
Comment (estebanlm) [reply src/Microdown-RichTextPresenter/MicMorphicTextAdapter.class.st:98]: I just adapted to what I have in SpMorphicTextAdapter. Is maybe useless, but it does not hurts.

File: src/NewTools-DocumentBrowser/MicDocumentBrowser.class.st (hunk 3 above)
Comment (estebanlm) [reply src/NewTools-DocumentBrowser/MicDocumentBrowser.class.st:321]: I will not change them :P

**Pull request merged**